src/RpcServer/RpcErrorCode.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2015-2023 The Neo Project.
+//
+// The Neo.Network.RPC is free software distributed under the MIT software license,
+// see the accompanying file LICENSE in the main directory of the
+// project or http://www.opensource.org/licenses/mit-license.php
+// for more details.
+//
+// Redistribution and use in source and binary forms with or without
+// modifications are permitted.
+
+using System.Collections.Generic;
+namespace Neo.Plugins;
```

Comment:
We should format the code as the rest of the code.

namespace x{ ... }

Reply:
this is the new .net 7.0 standard, x{..} is the old one, since we are already using .net 7.0, using new standard would be a proper choice.

Reply:
But the code style must be unified

Reply:
We don't need to waste time on these non functional issues. There are already tons of name style issues, comments issues, typo issues. Let's focus on fixing bugs please.

Reply:
Following the same coding style is important, we use `dotnet format` to detect some inconsistencies because of that.

Reply:
Following the same should not mean use net6.0 style in net7.0. A new C# standard is a new standard, when we create a new C# file, it is net.7.0 by default. dotnet format wont say its wrong format. If new coding style is considered `different coding style`, are we not going to use any new coding styles in the future?

Reply:
I did not do it on purpose, but IDEs generate new C# files with this style, its by standard now.

Reply:
I'm not against to this new style, I'm against to have 2 files different than the rest of the project.

Reply:
> I'm not against to this new style, I'm against to have 2 files different than the rest of the project.

Then how do you maintain a project that lasts for 10 or 20 years? with millions of lines of code written cross multiple programming language versions? for instance, we have code with style of C# 6, then we are working on C# 10, are you saying we either change millions of lines of C# 6.0 code into C# 10 or we just align to C# 6?

And i have no idea why the language standard suddenly became a thing for the project while the variable name has never attracted attention?

For instance, this few lines of code contains at lesat 3 different name style.
![Screenshot 2023-11-13 at 9 39 53 AM](https://github.com/neo-project/neo-modules/assets/10189511/239ec2ad-e817-49c2-a0f0-bdbd0d3192a3)

Reply:
What is coding style?

- Naming Conventions
- Bracing Style
- Spacing
- Commenting and Documentation
- Organize namespaces and classes logically
- Error Handling
- Principles for object-oriented design

Reply:
> Then how do you maintain a project that lasts for 10 or 20 years?

Run it through `dotnet format` (or whatever other tool) once, commit (with no other changes) and work off of this unified base?

Reply:
> > I'm not against to this new style, I'm against to have 2 files different than the rest of the project.
> 
> If you do care about coding style, here is the C# coding conventions https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#style-guidelines .
> 
> Then how do you maintain a project that lasts for 10 or 20 years? with millions of lines of code written across multiple programming language versions? for instance, we have code with style of C# 6, then we are working on C# 10, are you saying we either change millions of lines of C# 6.0 code into C# 10 or we just align to C# 6?
> 
> And i have no idea why the language standard suddenly became a thing for the project while the variable name has never attracted attention?
> 
> For instance, this few lines of code contains at least 3 different name styles, this is coding style, not language standard. See the yellow lines under the variable names? it indicate the name does not follow the C# coding style suggestion, that is coding style. ![Screenshot 2023-11-13 at 9 39 53 AM](https://user-images.githubusercontent.com/10189511/282342698-239ec2ad-e817-49c2-a0f0-bdbd0d3192a3.png)
> 
> How about we just focus on solving problems?

this must be solved, but introduce more different is not the way.

Reply:
>this is the new .net 7.0 standard, x{..} is the old one, since we are already using .net 7.0, using new standard would be a proper choice.

actually its C# 10 standard and they on C# 12 now, So its old. `dotnet` doesn't care only the compiler does.

Reply:
> > this is the new .net 7.0 standard, x{..} is the old one, since we are already using .net 7.0, using new standard would be a proper choice.
> 
> actually its C# 10 and they on C# 12 now, So its old. `dotnet` doesn't care only the compiler does.
> 
> https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-10

Not a problem for me now. Everyone agrees to use the traditional coding style, you better also update your code.